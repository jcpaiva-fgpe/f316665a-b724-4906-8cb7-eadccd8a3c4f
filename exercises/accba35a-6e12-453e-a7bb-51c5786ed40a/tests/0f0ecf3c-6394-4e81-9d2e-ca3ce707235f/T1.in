4
-1 4 5
7 7 7
4 3 6
8 2 1
