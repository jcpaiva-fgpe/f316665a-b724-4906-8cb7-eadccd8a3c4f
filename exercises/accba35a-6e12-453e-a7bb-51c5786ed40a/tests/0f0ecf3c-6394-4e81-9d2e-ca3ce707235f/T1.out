Claro que nao!
Sim, equilatero.
Sim.
Claro que nao!
